examples/others/observer/lv_example_observer_3.c

```diff
@@ -120,10 +120,10 @@ static void close_clicked_event_cb(lv_event_t * e)
 /*Watch all related subject to display the current time correctly*/
 static void time_observer_cb(lv_observer_t * observer, lv_subject_t * subject)
 {
-    int32_t hour = lv_subject_get_int(lv_subject_get_group_element(subject, 0));
-    int32_t minute = lv_subject_get_int(lv_subject_get_group_element(subject, 1));
-    int32_t format = lv_subject_get_int(lv_subject_get_group_element(subject, 2));
-    int32_t am_pm = lv_subject_get_int(lv_subject_get_group_element(subject, 3));
```

Comment:
`lv_subject_get_int` has `int32_t` return value. Why did it cause warning and when?

Reply:
It causes warning below in line 131. 
```
lv_label_set_text_fmt(label, "%d:%02d", hour, minute);
```

We can either cast it to `int` or stick to `LV_PRId32`. I choose the easier one since hour, minitue, etc. won't overflow the normal `int` value.

Reply:
I might cause an other warning about truncating the integer. (Especially in MSVC is picky about it)

I suggest using `LV_PRId32` to show this practice in the example.